CIP-0036/README.md

```diff
@@ -93,181 +136,460 @@ The intention with this design is that if projects beyond Catalyst implement thi
+
+1. Proves control of the Vote Key with a witness; and
+2. Proves control of the Stake Address with a witness; and
+3. Has a valid Payment address; and
+4. Has the highest nonce of any registration that has the same Vote Key; and
+5. Has the Purpose set to 0.
+
+These conditions are designed to inhibit shadow delegation.
+Shadow delegation is where a voter registers with another voters voting key, and does not participate.
+They do this simply to earn a voter reward, with no further effort on their behalf.
+In this case, anyone who shadow delegates will deprive themselves of both their voting power and any voter reward they may have enjoyed from voting.
+The voter who properly registered their voting key would be the sole beneficiary.
+Any voter who wishes to accumulate multiple wallets worth of voting power into their own controlled voting key may still do so, and they will receive the full voter reward they were entitled to.
+
+##### CIP-15 & CIP-36 Format Registrations
+
+If the same Voting key appears in both CIP-15 and CIP-36 registrations and at least one CIP-36 registration has witnessed to [Voting Key](#voters-public-key), then the CIP-15 registrations will be treated as unwitnessed voting keys as in [CIP-36 Only](#cip-15-format-registrations-only).
+
+Otherwise, the CIP-36 Registrations will be considered invalid, and the CIP-15 registrations will be treated as in [CIP-15 Only](#cip-15-format-registrations-only).
+
+#### Representative Voters
+
+Representatives MUST witness their voting key otherwise their representative voter registration will be INVALID.
+
+A Representative accumulates ALL of their own voting power associated their Stake Address, plus the portion of ADA delegated to them by a Voter.  Representatives are a different class of voter, and may have different rules associated with their voting and conduct.  For Example:
+
+- A Representative may have their votes de-anonymized at the end of a Catalyst Voting Event to prove their voting behaviour.
+- May Earn different rewards, depending on their status (Public or Private Representative).
+- May have different Voting Power Caps and Limits compared to a Self Registered Voter.
+- Other differences associated with the role.
+  - These differences are project Catalyst parameters and are outside of the scope of this document, except to highlight that Representatives are a different class and voter role than a self registered voter.
 
-Given the above, we choose to associate staking credentials with voting keys. At the moment, the only supported staking credential is a staking key. Since most Cardano wallets only use base addresses for Shelley wallet types, in most cases this should perfectly match the user's wallet.
+A Voter will earn to their reward address the voter reward their individual registration entitles them to.
+Voter rewards can only be accumulated if multiple registrations use the same payment address.
+
+#### Delegated Voters
```

Comment:
I think it is good to add this detail somewhere in this section too
`- Registrations with a weight of 0 are excluded from the array when the registration is processed`